setup.sh

```diff
@@ -10,14 +10,10 @@ if [[ $(cat /etc/os-release | grep -i debian) != "" ]]; then
   echo "Client is a Debian-based system. Installing binaries"; 
   echo
   echo "*** RUNNING APT-GET UPDATE ***"
-  sudo apt-get update --allow-releaseinfo-change
-  if [ $? -ne 0 ]; then echo "ERROR: 'apt-get update' failed with error code: $?"; exit 1; fi
-
-  echo
+  sudo apt update --allow-releaseinfo-change || true
```

Comment:
This will disable the error checking on apt update. If apt fails to update, then the installs may result in outdated packages. 

I feel like if there's an error running apt update, it should be addressed by the end user rather than ignoring it.

Reply:
If you force "|| true" though, wont the $? return 0? If that's the case then line 14 will never throw the error. 

On the other point, suppose a bad repository is accessed before the valid pikaraoke dependent-repositories are. The script would error AND the install would occur with out of date repositories. I don't see a way to handle this cleanly.